includes/js/bundle.js

```diff
@@ -20271,7 +20271,7 @@
 	'use strict';
```

Comment:
This file is generated on the fly right? If so, no need to commit the changes for this - keep it out of git version control.